target/classes/templates/start.html

```diff
@@ -15,11 +15,14 @@
 <h1>Fundraiser Manager</h1>
     <!---<div th:replace="fragments/topnav :: navbar">
     </div> -->
-    <div class="topnav">
+    <!--<div class="topnav">
         <a href="start.html">Home</a>
         <a href="members.html">Members</a>
         <a href="fundraisers.html">Fundraiser</a>
-    </div>
+    </div> -->

```

Comment:
Avoid committing commented-out code to version control, as version control offers better approaches:
1) If you are curious what used to be there, diff commits to see history.
2) If it is an experimental feature, use a feature branch to keep it separate from master.